packages/cli/src/commands/push.ts

```diff
@@ -35,12 +35,13 @@ type PushArgs = {
   destination?: string;
   branchName?: string;
   upsert?: boolean;
-  'batch-id'?: string;
+  'job-id'?: string;
```

Comment:
I'm not certain about renaming. Do we want to keep the old name for backward compatibility?

Reply:
yes.

Reply:
@RomanHotsiy should we also add `job-size` instead `batch-size` ?

Reply:
Let's keep batch size I think.